### ISSUE TITLE
Boosting Rate in Regression

### ISSUE THREAD

**Issue by PolarBearXBK** [sbadirli/GrowNet#4]
Hi,
I was wondering why in line 148 of GrowNet/Regression/main_reg_cv.py 
`loss = loss_f1(net_ensemble.boost_rate * out, grad_direction)  # T`
the boosting rate needed to be multiplied with the outputs of the current model? With traditional boosting ideas it seems like training should be done directly on residuals. without the boosting rate. Is there something here that I'm missing? Thanks!

**Comment by sbadirli** [comment]
Hi,

I am sorry for the late response, I am quite busy due to my PhD stuff right now. I will get back to you asap.

**Comment by PolarBearXBK** [comment]
No worries, thanks for your time and good luck with your PhD
